suave/gen/main.go

```diff
@@ -303,9 +303,22 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// List of suave precompile addresses
+var ( {{range .Functions}}{{.Name}}Addr = common.HexToAddress("{{.Address}}")
```

Comment:
The same thing is done in #40. Once both things are merged I will consolidate this in a single place.